src/shopping-component.js

```diff
@@ -4,6 +4,7 @@ import 'whatwg-fetch'; // eslint-disable-line import/no-unassigned-import
 import constants from './lib/constants';
 import { setupUserDetails } from './lib/user-configuration';
 import { shoppingAnalyticsSetup } from './lib/shopping-analytics';
+import { setupShoppingDDL, DDL_NAME } from './lib/ddl/shopping-ddl';
```

Comment:
What is DDL stands for?

Reply:
It is the naming convention that currently used by PPTM, `paypalDDL` . Not sure what it stands for though